src/QmlControls/CameraSettings.qml

```diff
@@ -0,0 +1,326 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtPositioning 5.14
+
+Rectangle {
+    id: root
+    color: '#000000'   // black
+
+    property var rootWidth: null;
+    property var rootHeight: null;
+
```

Comment:
The rootWidth and rootHeight properties are declared but not bound to any values. Consider binding these properties to the actual screen dimensions or relevant parent dimensions.

property int rootWidth: Screen.width
property int rootHeight: Screen.height